### ISSUE TITLE
Change hyperparameter tuning examples

### ISSUE THREAD

**Issue by erinmgraham** [carpentries-incubator/intro-image-classification-cnn#56]
Andrew: I think hyperparameter search is something important to be introduced as early as possible, but i’m not sure if some of the search space (activation functions) would be something they should look at checking first. Optimiser choice and learning rate are low hanging fruit in this regard.
